7_react/react1/week3/hw-react-week3/src/components/EditForm.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+const EditForm = ({ task, onCancel, onSave }) => {
+  const [editedDescription, setEditedDescription] = useState(task.description);
+  const [editedDueDate, setEditedDueDate] = useState(task.dueDate);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    onSave({
+      description: editedDescription,
+      dueDate: editedDueDate,
+    });
+  };
+
+  return (
+    <div className="editForm">
+      <form onSubmit={handleSubmit}>
+        <label>Description:</label>
```

Comment:
It's always a good idea to use the `for=` attribute on labels to connect them to the respective input.
This will help screen readers for people with visual impairment etc.
See here: https://www.w3schools.com/tags/tag_label.asp